deployment/CE02SHSP_Deploy.csv

```diff
@@ -62,39 +62,39 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
-11,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00353,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01519,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00871,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAD-00184,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00505,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00266,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
-11,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12886,2018-09-29T23:20:25,,44.637117,-124.302617,CGMCE-02SHSP-00011,SR1813,CGVEH-CSPPCE-00008,1,,,,,,80,
+8,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00003,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00346,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00308,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01206,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00870,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00138,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00411,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00282,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+8,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-11779,2017-12-07T19:01:00,2018-01-02T18:35:00,44.64102,-124.30216,CGMCE-02SHSP-00008,PS1712A,CGVEH-CSPPCE-00003,1,,,,,80,80,
+9,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00006,2018-04-20T18:22:00,2018-05-09T21:00:00,44.63362,-124.302583,CGMCE-02SHSP-00009,PS201804A,CGVEH-CSPPCE-00006,1,,,,,87,87,
+9,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00349,2018-04-20T18:22:00,2018-05-09T21:00:00,44.63362,-124.302583,CGMCE-02SHSP-00009,PS201804A,CGVEH-CSPPCE-00006,1,,,,,87,87,
```

Comment:
These values should be 80, not 87. The Washington site is 87 meters.

Reply:
done.